coordinator/coordinator.go

```diff
@@ -823,6 +827,36 @@ func (c *Coordinator) runFeatureMaterializeJob(resID metadata.ResourceID, schedu
 		return fmt.Errorf("materialize feature register: %v", err)
 	}
 	c.Logger.Debugw("Resource Table Created", "id", featID, "schema", schema)
+
+	// TODO: refactor this into a separate method
+	if featureProvider.Type() == string(pt.DynamoDBOnline) {
+		c.Logger.Debugw("Feature provider is DynamoDB", "id", featID)
+		config := pc.DynamodbConfig{}
+		if err := config.Deserialize(featureProvider.SerializedConfig()); err != nil {
+			return fmt.Errorf("could not deserialize DynamoDB config due to error: %v", err)
+		}
+		if config.ImportFromS3 {
```

Comment:
should we also check that the offline store is `s3`?

Reply:
At this point in the materialization, I can't effectively do that; however, I can add a check in the runner factory to ensure we throw an exception if the file store isn't S3.